### PR TITLE
Hotfix/ADF-865/Return empty dependencies list if feature flag is disabled

### PR DESCRIPTION
**Pull request by shpran**
Backport for [ADF-865](https://oat-sa.atlassian.net/browse/ADF-865)

### CODE REVIEW

**Review by gabrielfs7 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "prefer-stable": true,
   "require": {
     "oat-sa/generis": "15.11.0",
-    "oat-sa/tao-core": "48.66.1",
+    "oat-sa/tao-core": "dev-hotfix/adf-865/user-cannot-remove-properties as 48.66.1",
```
Comment (gabrielfs7) [inline composer.json:31]: - [ ] Update to proper branch before merge

Comment (hectoras) [inline composer.json:31]: Question: Shouldn't we make a new tag `48.66.2` in `tao-core` with the backport and point to that here?

Comment (gabrielfs7) [reply composer.json:31]: We will at the moment of the release. January was not released yet, so we do not create tags yet :)

**Review by bziondik — APPROVED** [review]
LGTM

**Pull request merged**